docker/Dockerfile

```diff
@@ -59,35 +20,43 @@ RUN apt-get update && \
     libffi-dev \
     pkg-config \
     clang \
-    autoconf && \
-  rm -rf /var/lib/apt/lists/*
-
-RUN pip3 install --upgrade pip
-RUN pip3 install --upgrade setuptools
-RUN pip3 install --upgrade pytesseract
-RUN pip3 install --upgrade pygraphviz
-WORKDIR /home
-
+    autoconf \
+    libssl-dev
+RUN pip install -U pip setuptools
 RUN git clone https://github.com/buffer/libemu.git && \
   cd libemu && \
   autoreconf -v -i && \
   ./configure && \
   make install && \
-  cd .. && \
-  rm -rf libemu
+  cd ..
+RUN git clone https://github.com/buffer/thug.git

```

Comment:
In theory this can be avoided if we mount the thug root project folder during the build and replace it with a `COPY`. That is it for now, but happy to change according to the final decision you would take.

Reply:
As suggested I have switched to `COPY`